index.html

```diff
@@ -0,0 +1,695 @@
+                </a>
+            </ul>
+        </nav>
+    </header>
+
+    <!-- SECTION-2 -->
+
+    <div class="container">
+
+        <aside class="sidebar">
+            <nav>
+                <ul class="sidebar-list-group">
+
+                    <a href="#navigations-id">
+                        <li class="sidebar-list-item gray">Navigation</li>
+                    </a>
+                    <a href="#buttons-id">
+                        <li class="sidebar-list-item gray">Buttons</li>
+                    </a>
+                    <a href="#avatars-id">
+                        <li class="sidebar-list-item gray">Avatar</li>
+                    </a>
+                    <a href="#alerts-id">
+                        <li class="sidebar-list-item gray">Alert</li>
+                    </a>
+                    <a href="#badges-id">
+                        <li class="sidebar-list-item gray">Badges</li>
+                    </a>
+                    <a href="#images-id">
+                        <li class="sidebar-list-item gray">Images</li>
+                    </a>
+                    <a href="#cards-id">
+                        <li class="sidebar-list-item gray">Cards</li>
+                    </a>
+                    <a href="#forms-id">
+                        <li class="sidebar-list-item gray">Forms</li>
+                    </a>
+                    <a href="#toasts-id">
+                        <li class="sidebar-list-item gray">Toast</li>
+                    </a>
+                    <a href='#modals-id'>
+                        <li class="sidebar-list-item gray">Modal</li>
+                    </a>
+                    <a href='#typographys-id'>
+                        <li class="sidebar-list-item gray">Typography</li>
+                    </a>
+                    <a href='#listgroups-id'>
+                        <li class="sidebar-list-item gray">List Group</li>
+                    </a>
+                </ul>
+            </nav>
+        </aside>
+
+        <main class="main">
+            <!-- NAVABAR -->
+            <div class="component" id="navigations-id">
+
+                <!-- FIRST-NAVBAR -->
+                <h1>Navigation</h1>
+                <p class="lead">Website navigation allows visitors to flow from one page to another</p>
+
+                <nav class="nav nav-dark">
+                    <h3>Navbar</h3>
+                    <button class="toggle" onclick="toggle('nav-list')">
+                        <img src="assets/Favicon/Hamburger_icon_white.svg" />
+                    </button>
+                    <ul class="nav-list" id='nav-list'>
+                        <li>Login</li>
+                        <li>SignUp</li>
+                        <li>Developers</li>
+                    </ul>
+                </nav>
+
+                <div class="display-none">
+                    <div onclick="showCode('navbar-filled')" class="pre-div">
```

Comment:
class pre-div defined but not used anywhere so can be removed if no plan to use it

Reply:
done